packages/design-system-docs/src/pages/startup/migrating-v2.md

```diff
@@ -3,7 +3,7 @@ title: Migrating to v2
 weight: 11
 ---
 
-CMS Design System v2 introduces several breaking changes, and this migration guide will outline high-level steps to upgrade to the latest version.
+The CMS Design System v2 release introduces several breaking changes, and this migration guide outlines high-level steps to upgrade. See the [release notes](https://github.com/CMSgov/design-system/releases/tag/core-2.0.0) for a detailed list of changes.
 
 ## NPM packages
 
```

Comment:
I'm wondering if we should break out the old dependencies with the new into 2 separate code blocks?

Previous dependencies
```
  "dependencies": {
    "@cmsgov/design-system-core": "^3.6.0",
    "@cmsgov/design-system-support": "^3.6.0",
    "@cmsgov/design-system-layout": "^3.6.0",
    ...
  }
```

New dependency
```
  "dependencies": {
    "@cmsgov/design-system": "^2.0.0",
    ...
  }
```